package.json

```diff
@@ -1351,6 +1360,11 @@
           "when": "view == confluent-schemas && confluent.schemaRegistrySelected",
           "group": "2_copy@1"
         },
+        {
+          "command": "confluent.schemas.copySchemaRegistrySubject",
+          "when": "view == confluent-schemas && confluent.schemaRegistrySelected",
+          "group": "2_copy@1"
+        },
```

Comment:
Unlike `confluent.schemas.copySchemaRegistryId`, which applies to the focused Schema Registry in the Schemas view, we don't want to expose a Subject-specific command at the top level under this `view/title` group. We _could_ if we needed, but that would then require showing a quickpick to the user to ask which Subject they want to use (since there isn't one specifically being focused).

Let's delete this block so we only expose the command/action when right-clicking a Subject from the Topics/Schemas views like you have further down in this file under the `view/item/context` section.